src/lib/MovingAverage.java

```diff
@@ -0,0 +1,36 @@
+package lib;
+
+public class MovingAverage {
+	private double[] readings;
+	private int sampleSize;
+	private int index = 0;
+	
+	/** 
+	 * Creates the average 
+	 * @param sampleSize size used to create the average   
```

Comment:
This is a bad parameter name and the javadoc doesn't make it clearer.  Anything in the MovingAverage class should help create the average.  Consider renaming to `numSamples` or similar.